doc/Learn/Mvux/FeedView.md

```diff
@@ -214,31 +231,41 @@ Example:
 
 ### ErrorTemplate
 
-The `FeedView` will display this template if an Exception was thrown by the underlying asynchronous operation.
-
-### UndefinedTemplate
+The `FeedView` will display this template if the underlying asynchronous operation throws an Exception.
 
-This template is displayed when the control loads before the underlying asynchronous operation has even been called.
-As soon as the asynchronous operation is invoked and awaited, the `FeedView` will switch to its `ProgressTemplate`, until the operation has resulted in data, which it will then switch to the `ValueTemplate`, or `NoneTemplate`, depending on the data result.
+Example:
 
-Typically, this template will only show for a very short period - a split second or so, depending on how long it takes for the page and its Model to load.
+```xml
+<FeedView ...>
+    <DataTemplate>
+        ...
+    </DataTemplate>
 
-## Other notable features
+    <ErrorTemplate>
+        <DataTemplate>
+            <TextBlock Text="An error has ocurred while loading the data..." />
+        </DataTemplate>
+    </ErrorTemplate>
+</FeedView>
+```
 
-### Refresh command property
+### UndefinedTemplate
 
-The `FeedView` provides an asynchronous `Command` you can bind to, which, when executed, will refresh the underlying `Feed` by re-requesting its data source.
+This template is displayed when the `FeedView` loads before the underlying asynchronous operation has even been called.
+As soon as the asynchronous operation is invoked and awaited, the `FeedView` will switch to its `ProgressTemplate` until the operation has resulted in data, at which point it will switch to the `ValueTemplate` or `NoneTemplate`, depending on the data result.
```

Comment:
```suggestion
This template is displayed when the `FeedView` loads before the underlying asynchronous operation has been called.
As soon as the asynchronous operation is invoked and awaited, the `FeedView` will switch to its `ProgressTemplate` until the operation results in data, at which point it will switch to the `ValueTemplate` or `NoneTemplate`, depending on the data result.
```